compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEVectorization.cpp

```diff
@@ -65,41 +66,37 @@ void AMDAIEVectorizationPass::runOnOperation() {
   SmallVector<Operation *> candidates;
   funcOp.walk([&](Operation *op) {
     // Only vectorize linalg ops (for now)
-    if (!isa<linalg::LinalgOp>(op)) return;
+    if (!isa<linalg::LinalgOp>(op)) return WalkResult::advance();
```

Comment:
Here and elsewhere : `WalkResult::skip()` perhaps ?

Reply:
I'm not sure, it might work (and would be better because more efficient in theory). But I'd prefer to play with this in another PR -- I'd like to try and get vectorization working for all of these ops eventually.